path_role.go

```diff
@@ -105,6 +108,11 @@ Defaults to 60 (1 minute) if set to 0 and can be disabled if set to -1.`,
 				Type:        framework.TypeCommaStringSlice,
 				Description: `Comma-separated list of 'aud' claims that are valid for login; any match is sufficient`,
 			},
+			"bound_claims_type": {
+				Type:        framework.TypeString,
+				Description: `How to interpret values in the map of claims/values (which must match for login): allowed values are string or glob`,
```

Comment:
```suggestion
				Description: `How to interpret values in the map of claims/values (which must match for login): allowed values are 'string' or 'glob'`,
```

Reply:
Implemented the change in a new commit (to simply reviewing it)